### ISSUE TITLE
MediaSource not supported on iPhone/iOS ... alternative approaches

### ISSUE THREAD

**Issue by josephaschmoe** [kvsur/openai-speech-stream-player#3]
I've tried getting streaming OpenAI TTS to work on iPhone/iOS, but mediasource is not supported.   So I don't believe this player will work in that case.   Am I wrong?   Alternative approaches?

**Comment by kvsur** [comment]
@josephaschmoe Yes bro, MediaSource is not fully supported in iOS Safari, you can find more from https://developer.mozilla.org/en-US/docs/Web/API/MediaSource#browser_compatibility.
This player was not tested on iOS devices during development. I'm sorry for that. I will try another way to make it working later. May be this month

**Comment by devanshSinghMadvit** [comment]
any update on this issue?

**Comment by naser-papi** [comment]
I need it on iOS also, is any new update guys?

**Comment by kvsur** [comment]
This bug fixed by Gemini3 Pro, looking detail for https://github.com/kvsur/openai-speech-stream-player/commit/bf588b8c16fb5f6cb044e3b9751921ea4f651278

**Comment by kvsur** [comment]
@josephaschmoe  @naser-papi @devanshSinghMadvit
